client/capability-token-2way/capability-token.4.x.rb

```diff
@@ -2,17 +2,17 @@
 require 'sinatra'
 
 get '/token' do
-    account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
-    auth_token = 'your_auth_token'
-    capability = Twilio::Util::Capability.new account_sid, auth_token
-    # Create an application sid at 
-    # twilio.com/console/phone-numbers/dev-tools/twiml-apps and use it here
-    capability.allow_client_outgoing "APXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
-    capability.allow_client_incoming params['ClientName']
-    token = capability.generate
+  account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
+  auth_token = 'your_auth_token'
+  capability = Twilio::Util::Capability.new account_sid, auth_token
+  # Create an application sid at
```

Comment:
What about capitalizing the acronym SID? I noticed you already did it with other examples.